app/helpers/transfers_helper.rb

```diff
@@ -1,2 +1,9 @@
 module TransfersHelper
+  def accountable_path(accountable)
+    if accountable.is_a?(Organization)
+      organization_path(accountable)
+    else
+      user_path(accountable.user)
+    end
+  end
```

Comment:
I'm not sure. I'd just do whatever it takes to follow Rails convention so that we don't have to define any path helper. But because we made this standout, we can work on it in an upcoming iteration, not here.